src/components/EventDateFilter/EventDateFilter.js

```diff
@@ -1,16 +1,45 @@
 import React, { PropTypes } from 'react';
 import DatePicker from 'react-datepicker';
+import moment from 'moment';
 
 import styles from './EventDateFilter.sass';
 
+const dateFormatToday = 'MMM D';
+const dateFormatMain = 'ddd MMM D';
+
+
+function isStartDateToday(startDate) {
+  const dateToday = moment();
+  return moment(startDate).isSame(dateToday, 'day');
+}
+
+function generateDateFormat(startDate) {
+  if (isStartDateToday(startDate)) {
+    return dateFormatToday;
+  }
+
+  return dateFormatMain;
```

Comment:
I don't totally follow what this is doing here -- can you add in a comment or two about why this check and different date formats are important?